filebeat/README.md

```diff
@@ -58,39 +53,62 @@ Give your logs some time to get from your system to ours, and then open [Logz.io
 
 Autodiscover allows you to adapt settings as changes happen. By defining configuration templates, the autodiscover subsystem can monitor services as they start running.
 
-#### 1. Store your Logz.io credentials
-Save your Logz.io shipping credentials as a Kubernetes secret.
-
-Replace `<<SHIPPING-TOKEN>>` with the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.
-
-Replace `<<LISTENER-HOST>>` with your region’s listener host (for example, `listener.logz.io`). For more information on finding your account’s region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html).
-
-Replace `<<CLUSTER-NAME>>` with your cluster's name.
-
-```shell
-kubectl create secret generic logzio-logs-secret \
-  --from-literal=logzio-logs-shipping-token=<<SHIPPING-TOKEN>> \
-  --from-literal=logzio-logs-listener=<<LISTENER-HOST>> \
-  --from-literal=cluster-name=<<CLUSTER-NAME>> \
-  -n kube-system
-```
-
-#### 2. Add logzio-k8s-logs repo to your helm repo list
+#### 1. Add logzio-k8s-logs repo to your helm repo list
 
 ```shell
 helm repo add logzio-helm https://logzio.github.io/logzio-helm/filebeat
 ```
 
 #### 3. Deploy
+In the following commands, make the following changes:
+* Replace `<<SHIPPING-TOKEN>>` with the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.
+
+* Replace `<<LISTENER-HOST>>` with your region’s listener host (for example, `listener.logz.io`). For more information on finding your account’s region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html).
+
+* Replace `<<CLUSTER-NAME>>` with your cluster's name.
+
 This daemonset's default autodiscover configuration is [hints based](https://www.elastic.co/guide/en/beats/filebeat/current/configuration-autodiscover-hints.html). If you wish to deploy it use:
 ```shell
-helm install --namespace=kube-system --set=configType=autodiscover logzio-k8s-logs logzio-helm/logzio-k8s-logs
+helm install --namespace=kube-system \
+--set configType='autodiscover' \
+--set secrets.logzioShippingToken='<<SHIPPING-TOKEN>>' \
+--set secrets.logzioListener='<<LISTENER-HOST>>' \
```

Comment:
I think we should pass region code instead (e.g us, eu, ca...) and the helm chart will do the rest

Reply:
Done!